src/main/resources/application.properties

```diff
@@ -13,3 +13,6 @@ fhir.terminology-server.url=
 
 # Medication / Condition rules spreadsheet
 rules.medication-condition.spreadsheet=CDS_Medication-Condition_Cards.xlsx
+
+# Medication / Medication rules csv
+rules.medication-medication.csv=CDS_Medication-Medication_Cards.csv
```

Comment:
All code and configuration files should end with a newline character for consistency and to allow git merging tooling to work properly.

Reply:
added new line